spec/javascripts/student_profile/NotesFeedPage.test.js

```diff
@@ -8,7 +8,9 @@ SpecSugar.withTestEl('high-level integration tests', function(container) {
       window.ReactDOM.render(<NotesFeedPage
         educatorsIndex={{}}
         eventNotes={[]}
-        eventNoteTypesIndex={{}} />, el);
+        eventNoteTypesIndex={{}}
+        onClickLoadMoreNotes={jest.fn()}
```

Comment:
@gabivoicu @alexsoble FYI, this missing prop was a warning in the Jest tests, so I fixed it up here.  Ideally we'd fixup the test setup so that it helps us catch this by actually fails the tests, I can look at that at some point too.

Reply:
@kevinrobinson Thanks for fixing that! I'm surprised it didn't fail the test.

Reply:
no problem!  Yeah I thought tests would catch this too, but I think when we switched over to the Jest test runner we lost that and hadn't noticed yet.